README.md

```diff
@@ -29,7 +29,7 @@ For NixOS, use the [NixPkg](https://github.com/NixOS/nixpkgs/blob/master/pkgs/ap
 - `--vertical-view` - shows the icon on the first line and temperature in a new line
 - `--custom-indicator STRING` - optional expression that will be shown instead of main indicator. current_conditions keys surrounded by {} can be used. For example, `"{ICON}{temp_C}({FeelsLikeC})"` will be transformed to `"text":"🌧️0(-4)"` in the output
 - `--lang LANG` - set language (currently `en`, `de`, `pl`, `tr`, `fr`, `ru`, `zh`, or `be`; submit a PR to add yours)
-
+- `--icon-family STRING` Changes Icon Family (Currently `noto`, `nerd`; submit a PR to add yours)
```

Comment:
I much rather have this as `--nerd-fonts`.

1. noto has nothing to do with wttrbar. we're using emojis by default. it's pure unicode. there are many other fonts (https://wiki.archlinux.org/title/fonts#Emoji_and_symbols) that have their own drawings for this unicode symbols and users are free to use them instead
2. I do not know of any other popular font system except nerd that is used for patching fonts with symbols, so it seems like making this "future-proof" by supporting other systems just makes the code more complex (e.g. error handling for not existing "font-families")